### DISCUSSION TITLE
How to recover from ErrorBoundary?

### DISCUSSION THREAD

**Discussion by zhaoyao91** [solidjs/solid#373] [Q&A] [answered]
With ErrorBoundary we can provide some error display to user. But we could add something like *retryButton* to allow user to recorver from error, but how to do this with ErrorBoundary? maybe provide a *rerender* function as an arg in fallback function?

**Comment by ryansolid** [comment, marked as answer]
Yeah that sounds reasonable I will look at adding a reset method to the callback.

**Reply by ryansolid** [reply]
Added in 0.24.14
